app.js

```diff
@@ -1,15 +1,25 @@
+const persist = require('choo-persist')
 const mount = require('choo/mount')
 const log = require('choo-log')
+const xtend = require('xtend')
 const choo = require('choo')
 
-const mainView = require('./pages/main')
+const opts = {
+  filter: (state) => {
+    state = xtend(state)
```

Comment:
1) It works in electron -> we don't need a polyfill
2) It will eventually work everywhere
3) It helps keep the dependency tree a bit simpler without extra lines of code
4) It will become the de facto standard for cloning - and I'd rather we're on the forefront of adoption
5) Better to teach and learn about it now than later
6) It's part of the programming language we're using